scripts/macos-setup.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
```

Comment:
I'd prefer that we keep this, even though it's just a wrapper. It's nice to be able to say 'run `scripts/macos-setup.sh` to get set up on mac'.

Reply:
sorry, I forgot about this one :o btw nobody should run this directly as this is not the only step to set up on mac. You must run the make command instead.

Reply:
oh wait, you commented on a deleted file